dali/pipeline/operators/python_function/dltensor_function.cc

```diff
@@ -77,8 +87,25 @@ void CopyDLTensorOutputs<CPUBackend>(HostWorkspace &ws, py::tuple &return_tuple)
   }
 }
 
+template <>
+void CopyDLTensorOutputs<GPUBackend>(DeviceWorkspace &ws, py::tuple &return_tuple, int batch_size) {
```

Comment:
There is some duplication between CopyDLTensorOutputs<CPUBackend> and CopyDLTensorOutputs<GPUBackend>. Can you extract some common part?

Reply:
done